### DISCUSSION TITLE
Adding Template for FAQ

### DISCUSSION THREAD

**Discussion by dihan** [mirceapiturca/Sections#3] [Q&A] [answered]
I'm not sure if its required but i've added a page.faq.liquid to the template folder with the following code to get FAQ to work

Thought it may help anyone else running to the same issue where sections cannot be added the pages

```
<div class="page-width">
  <div class="grid">
    <div class="grid__item medium-up--five-sixths medium-up--push-one-twelfth">
      <div class="section-header text-center">
        <h1>{{ page.title }}</h1>
      </div>

      <div class="rte">
        {% section 'faq' %}
      </div>
    </div>
  </div>
</div>
```

**Comment by mirceapiturca** [comment, marked as answer]
Hi @dihan 
Yes, that would be the "static" installation on a given page. This is not mentioned in the installation, thanks for adding this!
With the new Online Store 2.0 things may change. The installation on a JSON template will be a bit different. I will update the docs.
